README.md

```diff
@@ -13,6 +13,8 @@ We are iterating and looking for feedback and collaboration, so please [**let us
  - [Getting Started Guide](https://docs.amplify.aws/start/getting-started/setup/q/integration/flutter)
  - [Example Application](https://github.com/aws-amplify/amplify-flutter/tree/master/example)
  - [Roadmap/Provide Feedback](https://github.com/aws-amplify/amplify-flutter/issues/5)
+ 
+ ⚠️ **For upcoming breaking changes in 0.0.1-dev.7 please refer this [issue](https://github.com/aws-amplify/amplify-flutter/issues/274) for migration details.**
```

Comment:
Should it be 0.0.2?

Reply:
Makes sense. Will update to 0.0.2

Reply:
Apparently 0.0.2 is not helping, ^0.0.1-dev.6 is still bringing in 0.0.2-dev.1